bucket/gcc.json

```diff
@@ -1,18 +1,108 @@
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-isl-0.22.1-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-libiconv-1.16-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5680.0df6b89f-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-make-4.3-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-mpc-1.1.0-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-mpfr-4.0.2-2-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-windows-default-manifest-6.4-3-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-winpthreads-git-8.0.0.5680.0df6b89f-1-any.pkg.tar.xz",
+                "http://repo.msys2.org/mingw/i686/mingw-w64-i686-zlib-1.2.11-7-any.pkg.tar.xz"
+            ],
+            "hash": [
+                "234396201fb4078ef8463497c04151c7826f0313dcb4b40c9b82aea3a282e819",
+                "e06d7889a222182ac745a85435544dee951ba59ac29217a65236f3e9cbebd765",
+                "37dfa72246fd2478b991c93b5cce930b872420d6785fe7605458b472472134d8",
+                "181728706f67f9ed5683e96dab903d4f9414410464cd57df109fcc1741920431",
+                "8eea145827de9961fa5da0da3a215b5dbba6ed5dbf76aec77eda0b9473f69c82",
+                "a7125f82ea9df7def887d40ddd186e92d0dd08b6b8c51afe374b0df32781de9e",
+                "d5b2b3117b3127a719a2d2aa2b142b3f440d5b3cf207705f20c78c8d96615584",
+                "c1b3f4ea1a5c71b944bd0a71d2c2f6a36c38a5762a2b193bc4facf75c33d9c7a",
+                "3bf4783aed8e705fe9be132feced44d7097180d825367bc2b46fe45c39d2b701",
+                "1ab69f7d69898e09bedfadd40027eae7ec4723c19a2dffcf6b6a82be9d7f3288",
+                "7e6f75306aec44e3351aa1ae0adb9eed892d99dd552d564a903e9200474a0599",
+                "bbd79d5059f116e8f4b8dd26ca661f384eee1082b265389dbfa364d316eea334",
+                "a550b3412f88fe3e00e5b982a45aa850f728619dcd7095442106418544374066",
+                "49a9bd81fe265fd969618f9ffee9926a92193cc86d409cb1988f35a4dad3fe79",
+                "599a0276820e3d342d1c494c4506aaf79fbbbc2843bbec7aae5f22a1b71da284",
+                "7ba5ed9c5e535fdc619138278fff0567dad39b9251d5dc5c4d708ab6f345146e",
+                "39e07e61d739ba8f066605a109a19db397be6f7ddd81e5172f49ed253fdbe49f",
+                "56323bc39c7de0ff727915c09c4aaa25b8396efc0d7eda0006d5951bb6a6b983",
+                "e69339fda92b19469d5a4e0a1d5f50af8d8a420dc66f823d653ac8b536240493",
+                "addf6c52134027407640f1cbdf4efc5b64430f3a286cb4e4c4f5dbb44ce55a42"
+            ],
+            "pre_install": [
+                "Move-Item -Path \"$dir\\mingw32\\*\" -Destination \"$dir\"",
+                "Remove-Item -Path \"$dir\\mingw32\", \"$dir\\.*\""
```

Comment:
`.*`?
You want to remove dot files or type?

Reply:
Yes, some files like `.BUILDINFO`, `.MTREE`, `.PKGINFO`.